doc/modules/prediction.rst

```diff
@@ -0,0 +1,156 @@
+.. currentmodule:: sklearn.model_selection
+
+.. _cutoffclassifier:
+
+========================================================
+Tuning cut-off decision threshold for classes prediction
+========================================================
+
+Classifiers are predictive models: they use statistical learning to predict
+outcomes. The outcomes of a classifier takes two forms: a "soft" score for each
+sample in relation to each class, and a "hard" categorical prediction (i.e.
+class label). Soft predictions are obtained using :term:`predict_proba` or
+:term:`decision_function` while hard predictions are obtained using
+:term:`predict`.
+
+In scikit-learn, there is a connection between soft and hard prediction. In the
+case of a binary classification, hard predictions are obtained by associating
+the positive class with probability value greater than 0.5 (obtained with
+:term:`predict_proba`) or decision function value greater than 0 (obtained with
+:term:`decision_function`).
```

Comment:
```suggestion
For binary classification, class labels are obtained by associating
the positive class with a score greater than 0.5 (obtained with
:term:`predict_proba`) or decision function value greater than 0 (obtained with
:term:`decision_function`).
```

Reply:
Maybe better not use the term "score" for a predicted probability.